knox/views.py

```diff
@@ -15,6 +18,12 @@ class LoginView(APIView):
     permission_classes = (IsAuthenticated,)
 
     def post(self, request, format=None):
+        if knox_settings.TOKEN_LIMIT_PER_USER is not None:
+            if request.user.auth_token_set.filter(expires__gt=timezone.now()).count() >= knox_settings.TOKEN_LIMIT_PER_USER:
+                return Response(
+                    {"error": "Maximum amount of tokens allowed per user exceeded."},
+                    status=status.HTTP_401_UNAUTHORIZED
```

Comment:
To me it feels more like a 403, @sphrak 
What do you think? https://en.wikipedia.org/wiki/List_of_HTTP_status_codes

Reply:
I agree @belugame I will fix your proposed changes in upcoming commit.